src/main/java/com/bootcamp/demo/restapi/exceptionhandler/ApiError.java

```diff
@@ -0,0 +1,61 @@
+package com.bootcamp.demo.restapi.exceptionhandler;
+
+import java.util.Objects;
+
+public class ApiError {
+
+    private int status;
```

Comment:
Considering http status code is already returned, does this add value? 
You could stick with a simpler version for exception handling.